packages/builder/src/build.js

```diff
@@ -141,7 +141,9 @@ async function copyManifest(manifestJSON) {
     }
 
     copy.commands = manifestJSON.commands.map(command => {
-      const script = command.script.replace(/\.(?!js)|\//g, '_')
+      const script = command.script
+        .replace(/\.(?!j|tsx?$)|\//g, '_')
+        .replace(/j|tsx?$/, 'js')
```

Comment:
The regex has a huge bug. It will replace `.js` to `.jss`！The correct one is `/[jt]sx?$/`.

```js
//output: a.jss
'a.js'.replace(/j|tsx?$/, 'js');
```

Reply:
OK, I see #284 have fixed it~ Bug It has another bug. It just replace command filename in manifest.json, but the file output name is the old style so that plugin can't find command file.

Reply:
I have created #285 pull request to fix another problem

Reply:
@lizheming thank you very much!